src/shared/tpm2-util.c

```diff
@@ -2048,33 +2044,57 @@ int tpm2_seal(const char *device,
         if (r < 0)
                 return r;
 
+        TPMI_ALG_HASH pcr_bank = 0;
+        if (hash_pcr_mask | pubkey_pcr_mask) {
```

Comment:
please write as `(hash_pcr_mask | pubkey_pcr_mask) != 0` to match our coding style, as this is not boolean (or boolean like, or a pointer), but can have arbitrary values